src/ActiveLogin.Authentication.BankId.AspNetCore/Areas/BankIdAuthentication/Views/BankId/_spinner.cshtml

```diff
@@ -0,0 +1,62 @@
+﻿<svg class="status-icon" width="150px" height="150px" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 100 100" preserveAspectRatio="xMidYMid" style="background: none; shape-rendering: auto;">
```

Comment:
I ran the SVG [through a minifier](https://jakearchibald.github.io/svgomg/) to make it slightly more compact. Also changed default color to black. Please consider using this one.


```
<svg class="status-icon" width="150" height="150" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" preserveAspectRatio="xMidYMid" style="background:0 0"><g><rect x="44" y="4" rx="66" ry="6" width="12" height="12" fill="#000000" class="spinner-component"><animate attributeName="opacity" values="1;0" keyTimes="0;1" dur="1s" begin="-0.9166666666666666s" repeatCount="indefinite"/></rect></g><g transform="rotate(30 50 50)"><rect x="44" y="4" rx="66" ry="6" width="12" height="12" fill="#000000" class="spinner-component"><animate attributeName="opacity" values="1;0" keyTimes="0;1" dur="1s" begin="-0.8333333333333334s" repeatCount="indefinite"/></rect></g><g transform="rotate(60 50 50)"><rect x="44" y="4" rx="66" ry="6" width="12" height="12" fill="#000000" class="spinner-component"><animate attributeName="opacity" values="1;0" keyTimes="0;1" dur="1s" begin="-0.75s" repeatCount="indefinite"/></rect></g><g transform="rotate(90 50 50)"><rect x="44" y="4" rx="66" ry="6" width="12" height="12" fill="#000000" class="spinner-component"><animate attributeName="opacity" values="1;0" keyTimes="0;1" dur="1s" begin="-0.6666666666666666s" repeatCount="indefinite"/></rect></g><g transform="rotate(120 50 50)"><rect x="44" y="4" rx="66" ry="6" width="12" height="12" fill="#000000" class="spinner-component"><animate attributeName="opacity" values="1;0" keyTimes="0;1" dur="1s" begin="-0.5833333333333334s" repeatCount="indefinite"/></rect></g><g transform="rotate(150 50 50)"><rect x="44" y="4" rx="66" ry="6" width="12" height="12" fill="#000000" class="spinner-component"><animate attributeName="opacity" values="1;0" keyTimes="0;1" dur="1s" begin="-0.5s" repeatCount="indefinite"/></rect></g><g transform="rotate(180 50 50)"><rect x="44" y="4" rx="66" ry="6" width="12" height="12" fill="#000000" class="spinner-component"><animate attributeName="opacity" values="1;0" keyTimes="0;1" dur="1s" begin="-0.4166666666666667s" repeatCount="indefinite"/></rect></g><g transform="rotate(210 50 50)"><rect x="44" y="4" rx="66" ry="6" width="12" height="12" fill="#000000" class="spinner-component"><animate attributeName="opacity" values="1;0" keyTimes="0;1" dur="1s" begin="-0.3333333333333333s" repeatCount="indefinite"/></rect></g><g transform="rotate(240 50 50)"><rect x="44" y="4" rx="66" ry="6" width="12" height="12" fill="#000000" class="spinner-component"><animate attributeName="opacity" values="1;0" keyTimes="0;1" dur="1s" begin="-0.25s" repeatCount="indefinite"/></rect></g><g transform="rotate(270 50 50)"><rect x="44" y="4" rx="66" ry="6" width="12" height="12" fill="#000000" class="spinner-component"><animate attributeName="opacity" values="1;0" keyTimes="0;1" dur="1s" begin="-0.16666666666666666s" repeatCount="indefinite"/></rect></g><g transform="rotate(300 50 50)"><rect x="44" y="4" rx="66" ry="6" width="12" height="12" fill="#000000" class="spinner-component"><animate attributeName="opacity" values="1;0" keyTimes="0;1" dur="1s" begin="-0.08333333333333333s" repeatCount="indefinite"/></rect></g><g transform="rotate(330 50 50)"><rect x="44" y="4" rx="66" ry="6" width="12" height="12" fill="#000000" class="spinner-component"><animate attributeName="opacity" values="1;0" keyTimes="0;1" dur="1s" begin="0s" repeatCount="indefinite"/></rect></g></svg>
```

Reply:
I changed to use this minified svg instead.